src/components/Token/Card.tsx

```diff
@@ -82,7 +82,7 @@ export const TokenCard: FC<Props> = ({ token }) => {
         <div className="card-actions">
           <button 
             disabled={loading}
-            className="btn btn-primary z-10"
+            className="btn btn-primary z-10 cursor-pointer"
```

Comment:
cursor-pointer is unnecessary since the btn class provides this styling already. can we please have this removed. once done, this PR is good to go!

Reply:
Great catch!!!  It's updated.  I will be making an additional bug and feature enhancement changes later on.  I just want to get my feet wet a little bit.  I am new to T3 stack, Web 3 and Base and eager to learn.  Thank you so much for giving me an opportunity!